app_dart/lib/src/foundation/utils.dart

```diff
@@ -104,6 +106,27 @@ Future<List<LuciBuilder>> getRepoBuilders(HttpClientProvider branchHttpClientPro
       .toList();
 }
 
+/// Returns LUCI builders based on [bucket].
```

Comment:
How is this different from the previous function? from docs they look like the same. Can we update docs or even better merge the two functions?

Reply:
Merged. It is more concise now.